app/views/books/show.html.erb

```diff
@@ -14,15 +14,15 @@
     <% unless @book.reviews.empty? %>
       <section class="top-reviews">
         <p>Top Reviews</p>
-        <% @book.grab_reviews('top', 3).each do |review| %>
-          <p>Title: <%= review.title %>, Rating: <%= review.rating %>, User: <%= link_to review.username %></p>
+        <%= @book.grab_reviews('top', 3).each do |review| %>
```

Comment:
Remove this erb equal sign too, I got it on a previous PR

Reply:
Removed per request, I might have missed these when I handled a merge conflict.